src/tz.cpp

```diff
@@ -3516,7 +3625,12 @@ init_tzdb()
 
     for (const auto& filename : files)
     {
-        std::ifstream infile(path + filename);
+        std::string file_path = path + filename;
+        if (!file_exists(file_path)) {
+          continue;
+        }
+        file_streambuf inbuf(file_path);
+        std::istream infile(&inbuf);
         while (infile)
```

Comment:
I think this previously relied on `infile` returning `false` if the file didn't exist and it couldn't be opened.

Now we have to explicitly check for file existence earlier on to avoid an error.

The list of files we look through are a few lines up in `CONSTDATA char*const files[]`, and some of those don't exist anymore in the up to date time zone data files so they just get skipped.